presto-docs/src/main/sphinx/connector/redis.rst

```diff
@@ -51,6 +51,9 @@ Property Name                       Description
 ``redis.hide-internal-columns``     Controls whether internal columns are part of the table schema or not
 ``redis.database-index``            Redis database index
 ``redis.password``                  Redis server password
```

Comment:
Few properties mention redis database, and this mentions redis server. Can we make them uniform?

Reply:
not really, user and password are used for authentication. Basically we are using database and server interchangeably. Anyways this is existing documentation, so let us keep it as is for now.